app/views/layouts/application.html.haml

```diff
@@ -16,3 +16,4 @@
     = yield
     %script{ :src => "https://code.jquery.com/jquery-2.1.1.min.js", :type => "text/javascript" }
     %script{ :src => "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.0/jquery.min.js" }
+    %script{:src => "//cdn.tinymce.com/4/tinymce.min.js"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace